health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/NavigationLogicWrapper.js

```
@@ -0,0 +1,805 @@
+                                                                        marginLeft: "auto",
+                                                                        display: "flex",
+                                                                        alignItems: "center",
+                                                                        gap: "0.25rem",
+                                                                        cursor: "pointer",
+                                                                    }}
+                                                                    onClick={() => removeCondition(editorIndex, idx)}
+                                                                    title={removeConditionLabel}
+                                                                    aria-label={removeConditionLabel}
+                                                                    role="button"
+                                                                >
+                                                                    <SVG.Delete fill={"#C84C0E"} width={"1.25rem"} height={"1.25rem"} />
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)